### DISCUSSION TITLE
Is SECRET_KEY_BASE in a docker-compose.yml file the same as SUPABASE_SECRET_KEY?

### DISCUSSION THREAD

**Discussion by martineboh** [supabase/supabase#489] [Questions] [answered]
I am wondering if the **SECRET_KEY_BASE** as found in the **supabase/realtime** `docker-compose.yml` file the same as SUPABASE_SECRET_KEY found the nextjs example:

```
const supabase = createClient(
  process.env.NEXT_PUBLIC_SUPABASE_URL,
  process.env.SUPABASE_SECRET_KEY
);
```

**Comment by soedirgo** [comment, marked as answer]
Hey @martineboh, they're different - `SECRET_KEY_BASE` is required by the Phoenix server in supabase/realtime, and in this context, they aren't useful. You just need to supply a value there.

If you're curious about what it is: https://elixirforum.com/t/can-someone-explain-config-prod-secret-exs-to-me/3810/2
